QEfficient/transformers/models/modeling_auto.py

```
@@ -380,6 +380,33 @@ def compile(
                 aic_num_cores=num_cores,
                 **compiler_options,
             )
+
+        # Construct the qconfig json file
+        huggingface_config = self.model.config.__dict__
+        pytorch_transforms = [cls.__name__ for cls in self._pytorch_transforms]
+        onnx_transforms = [cls.__name__ for cls in self._onnx_transforms]
+        try:
```

remove try-except let it fail in case of any exception